udata/core/metrics/__init__.py

```diff
@@ -111,10 +112,12 @@ def iso(self, value):
                 'Unsupported format: {0} ({1})'.format(value, type(value)))
 
     def trigger_update(self):
-        self.need_update.send(self)
+        if not current_app.config['TESTING'] or current_app.config.get('FORCE_METRICS'):
+            self.need_update.send(self)
 
     def notify_update(self):
-        self.updated.send(self)
+        if not current_app.config['TESTING'] or current_app.config.get('FORCE_METRICS'):
```

Comment:
Instead of testing both `TESTING` and `FORCE_METRICS`, I think it would be cleaner to have a single `USE_METRICS` defined to `True` in `udata.settings.Defaults` and overriden to `False` in `udata.settings.Testing` so:
- we avoid runtime code to perform to have a view on testing
- we have all testing parameters in the same place
- activating metrics in test is simply setting `config.USE_METRICS` to True
@davidbgk I think local load of backups is not performed inside the application so metrics are not triggered.